### PR TITLE
Add `util.getMajorMinorVersion` method

### PR DESCRIPTION
**Pull request by radtriste**

### CODE REVIEW

File: vars/util.groovy [hunk 1]
```diff
@@ -157,17 +157,17 @@ def isProjectTriggeringJob(def projectGroupName) {
  */
 def getProjectTriggeringJob() {
     if (!env.ghprbGhRepository && !env.GIT_URL) {
-        throw new Exception("There's no way to get which project is triggering the job");
+        throw new Exception("There's no way to get which project is triggering the job")
```
Comment (Ginxo) [inline vars/util.groovy:160]:
```suggestion
        throw new Exception('There's no way to get which project is triggering the job')
```

File: vars/util.groovy [hunk 2]
```diff
@@ -259,6 +259,16 @@ Integer[] parseVersion(String version) {
     }
 }
 
+String getMajorMinorVersion(String version) {
+    try {
+        String[] versionSplit = version.split("\\.")
```
Comment (Ginxo) [inline vars/util.groovy:264]:
```suggestion
        String[] versionSplit = version.split('\\.')
```

File: vars/util.groovy [hunk 3]
```diff
@@ -276,18 +286,18 @@ String calculateTargetReleaseBranch(String currentReleaseBranch, int addToMajor
         targetBranch = "${newMajor}.${newMinor}.${versionSplit[2]}"
     } else {
         println "Cannot parse targetBranch as release branch so going further with current value: ${targetBranch}"
-    }
+        }
```
Comment (Ginxo) [inline vars/util.groovy:289]:
```suggestion
   }
```

File: vars/util.groovy [hunk 4]
```diff
@@ -276,18 +286,18 @@ String calculateTargetReleaseBranch(String currentReleaseBranch, int addToMajor
...
 /**
  * It prepares the environment to avoid problems with plugins. For example files from SCM pipeline are deleted during checkout
  */
 def prepareEnvironment() {
-    println """
+    println '''
 [INFO] Preparing Environment
 [INFO] Copying WORKSPACE content env folder
-    """
+    '''
```
Comment (Ginxo) [inline vars/util.groovy:300]:
```suggestion
'''
```

File: vars/util.groovy [hunk 5]
```diff
@@ -533,14 +543,14 @@ ${failedTest.details ?: failedTest.stacktrace}
 """
         } catch (err) {
             echo 'No test results found'
-        }
+}
```
Comment (Ginxo) [inline vars/util.groovy:546]:
```suggestion
        }
```

**Review by Ginxo — APPROVED** [review]
LGTM, just few minor technical suggestions

File: vars/util.groovy (hunk 2 above)
Comment (radtriste) [reply vars/util.groovy:264]: This is a tricky one so I won't change like that

**Comment by radtriste** [comment]
I removed the format commit. It created more problems than solving some ... ^^

**Pull request merged**